File: docs/examples/csharp/Kafka/Consumer/Meters.cs
namespace Consumer
{
    //const string createTable = "CREATE STABLE meters (ts timestamp, current float, voltage int, phase float) TAGS (location binary(64), groupId int);";

    public class MeterTag
    {
        public string Location { get; set; }
        public int GroupID { get; set; }

        public MeterTag(string location, int groupId)
        {
            Location = location;
            GroupID = groupId;
        }
        public override string ToString()
        {
            return $"location:{Location},GroupID:{GroupID}";
        }
    }

    public class MeterValues
    {
        public long TimeStamp { get; set; }
        public float Current { get; set; }

        public int Voltage { get; set; }
        public float Phase { get; set; }

        public MeterValues(long ts, float current, int voltage, float phase)
        {
            TimeStamp = ts;
            Current = current;
            Voltage = voltage;
            Phase = phase;
        }

        public override string ToString()
        {
            return $"{TimeStamp},{Current},{Voltage},{Phase}";
        }
    }
}

File: docs/examples/JDBC/JDBCDemo/src/main/java/com/taos/example/Meters.java
package com.taos.example;

import java.sql.Timestamp;

public class Meters {
    private Timestamp ts;
    private float current;
    private int voltage;
    private int groupid;
    private String location;

    public Timestamp getTs() {
        return ts;
    }

    public void setTs(Timestamp ts) {
        this.ts = ts;
    }

    public float getCurrent() {
        return current;
    }

    public void setCurrent(float current) {
        this.current = current;
    }

    public int getVoltage() {
        return voltage;
    }

    public void setVoltage(int voltage) {
        this.voltage = voltage;
    }

    public int getGroupid() {
        return groupid;
    }

    public void setGroupid(int groupid) {
        this.groupid = groupid;
    }

    public String getLocation() {
        return location;
    }

    public void setLocation(String location) {
        this.location = location;
    }

    @Override
    public String toString() {
        return "Meters{" +
                "ts=" + ts +
                ", current=" + current +
                ", voltage=" + voltage +
                ", groupid=" + groupid +
                ", location='" + location + '\'' +
                '}';
    }
}
